.github/workflows/ci.yml

```diff
@@ -95,19 +95,19 @@ jobs:
     container:
       image: rust
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v5
       - uses: dtolnay/rust-toolchain@stable
         with:
           targets: x86_64-unknown-linux-gnu
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v6
         with:
-          node-version: '16'
+          node-version: '25'
```

Comment:
We should change this (and other similar node-version entries) to `20`. It's the oldest one we can support at this moment and we should try to keep this as backwards compatible as possible.

Reply:
Updated! Will merge after CI passes.